Makefile

```
@@ -29,11 +29,34 @@ LDFLAGS := -s -w -extldflags '-static'
 
 SRCFILES := $(shell find . ! -path './out/*' ! -path './.git/*' -type f)
 
+ALL_ARCH=amd64 arm arm64 ppc64le s390x
+ML_PLATFORMS=linux/amd64,linux/arm,linux/arm64,linux/ppc64le,linux/s390x
 ALL_BINARIES ?= $(addprefix out/configmap-reload-, \
-									$(addprefix linux-,amd64 ppc64le s390x arm64) \
+									$(addprefix linux-,amd64 ppc64le s390x arm arm64) \
 									darwin-amd64 \
 									windows-amd64.exe)
 
+ifeq ($(GOARCH),amd64)
+	BASEIMAGE?=busybox
+	BINARY=configmap-reload-linux-amd64
```

You can just move the `BINARY` variable outside of the `ifeq` blocks as:

```suggestion
BINARY=configmap-reload-linux-$(GOARCH)
```